R/bs-theme.R

```diff
@@ -97,6 +97,9 @@
 #' @param base_font The default typeface.
 #' @param code_font The typeface to be used for code. Be sure this is monospace!
 #' @param heading_font The typeface to be used for heading elements.
+#' @param font_scale A scalar multiplier to apply to the base font size. For
+#'   example, a value of `1.5` increases font sizes by 150% and a value of `0.8`
+#'   decreases by 80%. Must be a positive number.
```

Comment:
I have a hard time working with percentages. Increasing by 150%, to me, says that it should go from 8 to (8 + 1.5 * 8) = 20. 

(And it is not decreasing by 80%. 10 -> 2)

```suggestion
#'   example, a value of `1.5` scales font sizes to 150% and a value of `0.8`
#'   scales to 80%. Must be a positive number.
```